draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -331,25 +329,25 @@
 
         <t>
                 The trigger resource has a state attribute.  
-                dCDN creates new triggers in "pending" state. Once dCDN starts processing a trigger, the trigger 
+                dCDN creates new triggers in the "pending" state. Once dCDN starts processing a trigger, the trigger 
                 state is set to "active". 
                 Once the trigger processing is complete, the state is set to either "complete" or "failed", 
                 depending on the processing outcome.
         </t>
         <t>
-                uCDN MAY request dCDN to cancel a trigger. If such request is accepted, the trigger state is changed to 
+                uCDN MAY request dCDN to cancel a trigger. If such a request is accepted, the trigger state is changed to 
                 "cancelling", and when the cancellation is complete, the trigger state changes to "cancelled".
         </t>        
         <t>         
-                For full description of the trigger resource, please refer to <xref target="ci-trigger-resource"/>.
+                For a full description of the trigger resource, please refer to <xref target="ci-trigger-resource"/>.
         </t>
 
    </section>
    <section title="Trigger Collection" anchor="model-for-cdni-triggers-trigger-collection-resource">
         <t>
-        Trigger collection is a resource that represents all trigger resources, belonging to a particular uCDN.
+        A trigger collection is a resource that represents all trigger resources, belonging to a particular uCDN.
         dCDN maintains one trigger collection resource for each uCDN. 
-        Each uCDN only has access to its own collection resource.
+        Each uCDN only has access to its collection resource.
```

Comment:
While still grammatically correct, I think the presence of the word "own" here helped emphasize that the uCDN does not have access to the collection resources of other uCDNs.  Without this word, I feel like I have to reread this sentence to determine that "its" should be emphasized instead.

Reply:
it's a particular style idiosyncrasy that Grammarly has